ifrs17/Import/Importers.ipynb

```diff
@@ -622,6 +648,70 @@
             "execution_count": 0,
             "outputs": []
         },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Yield Curves "
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "Import.DefineFormat(ImportFormats.YieldCurve, async (options, dataSet) => {",
+                "\n    Activity.Start();",
+                "\n    var args = await GetArgsFromMainAsync(dataSet);",
+                "\n    if(ApplicationMessage.HasErrors())  return Activity.Finish();",
+                "\n",
+                "\n    var workspace = Workspace.CreateNew();",
+                "\n    workspace.Initialize(x => x.FromSource(DataSource).DisableInitialization<RawVariable>().DisableInitialization<IfrsVariable>());",
+                "\n    var oldYieldCurves = await DataSource.Query<YieldCurve>().ToArrayAsync();",
+                "\n",
+                "\n    var importLog = await Import.FromDataSet(dataSet)",
+                "\n        .WithType<YieldCurve>((dataset, datarow) => {",
+                "\n            var currency = datarow.Field<string>(nameof(YieldCurve.Currency));",
+                "\n            var values = datarow.Table.Columns.Where(c => c.ColumnName.StartsWith(nameof(YieldCurve.Values))).OrderBy(c => c.ColumnName.Length).ThenBy(c => c.ColumnName)",
+                "\n                                .Select(x => datarow.Field<string>(x.ColumnName).CheckStringForExponentialAndConvertToDouble()).ToArray();",
+                "\n            var item = new YieldCurve {",
```

Comment:
YieldCurve have a property string called Name. 
Expect the columnName to be Name.